### ISSUE TITLE
kubelet not work in raspbian version 1.20.0.

### ISSUE THREAD

**Issue by oldthreefeng** [kubernetes/kubernetes#97646]
**What happened**:
kubelet not work 

**What you expected to happen**:
kubelet work well 

**How to reproduce it (as minimally and precisely as possible)**:
download file from offical tar ball .
binary is download from :
https://dl.k8s.io/v1.20.0/kubernetes-server-linux-arm64.tar.gz

**Anything else we need to know?**:

or should i download  https://dl.k8s.io/v1.20.0/kubernetes-server-linux-arm.tar.gz ?

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

here is the env.
```
root@destiny:~/kube/bin# uname  -a
Linux destiny 5.4.51-v8+ #1333 SMP PREEMPT Mon Aug 10 16:58:35 BST 2020 aarch64 GNU/Linux

root@destiny:~/kube/bin# ./kubelet version
-bash: ./kubelet: No such file or directory

root@destiny:~/kube/bin# ./kubectl  version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.0", GitCommit:"af46c47ce925f4c4ad5cc8d1fca46c7b77d13b38", GitTreeState:"clean", BuildDate:"2020-12-08T17:59:43Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/arm64"}
The connection to the server localhost:8080 was refused - did you specify the right host or port?
root@destiny:~/kube/bin# file kubelet
kubelet: ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), dynamically linked, interpreter /lib/ld-linux-aarch64.so.1, for GNU/Linux 3.7.0, BuildID[sha1]=1b6d34119bb0682bc37493e38c874dde0c722fa8, stripped


root@destiny:~/kube/bin# cat /etc/os-release
PRETTY_NAME="Raspbian GNU/Linux 10 (buster)"
NAME="Raspbian GNU/Linux"
VERSION_ID="10"
VERSION="10 (buster)"
VERSION_CODENAME=buster
ID=raspbian
ID_LIKE=debian
HOME_URL="http://www.raspbian.org/"
SUPPORT_URL="http://www.raspbian.org/RaspbianForums"
BUG_REPORT_URL="http://www.raspbian.org/RaspbianBugs"

root@destiny:~/kube/bin# cat /proc/cpuinfo

processor       : 3
BogoMIPS        : 108.00
Features        : fp asimd evtstrm crc32 cpuid
CPU implementer : 0x41
CPU architecture: 8
CPU variant     : 0x0
CPU part        : 0xd08
CPU revision    : 3

Hardware        : BCM2835
Revision        : d03114
Serial          : 100000003625f63c
Model           : Raspberry Pi 4 Model B Rev 1.4

```

**Comment by oldthreefeng** [comment]
when doing kubeadm init cause this problem.
```
root@destiny:~# free -h
              total        used        free      shared  buff/cache   available
Mem:          7.6Gi       182Mi       3.9Gi       439Mi       3.6Gi       6.9Gi
Swap:            0B          0B          0B
root@destiny:~# kubeadm init --kubernetes-version=1.20.1
[init] Using Kubernetes version: v1.20.1
[preflight] Running pre-flight checks
        [WARNING IsDockerSystemdCheck]: detected "cgroupfs" as the Docker cgroup driver. The recommended driver is "systemd". Please follow the guide at https://kubernetes.io/docs/setup/cri/
        [WARNING SystemVerification]: this Docker version is not on the list of validated versions: 20.10.1. Latest validated version: 19.03
        [WARNING SystemVerification]: missing optional cgroups: hugetlb
error execution phase preflight: [preflight] Some fatal errors occurred:
        [ERROR Mem]: the system RAM (1 MB) is less than the minimum 1700 MB
[preflight] If you know what you are doing, you can make a check non-fatal with `--ignore-preflight-errors=...`
To see the stack trace of this error execute with --v=5 or higher
```

**Comment by aojea** [comment]
This seems a duplicate of https://github.com/kubernetes/kubeadm/issues/2365

**Comment by neolit123** [comment]
just skip the preflight check with --ignore-preflight-errors=Mem
the fix will be in 1.21.

**Comment by oldthreefeng** [comment]
@neolit123 
how about question 1 . Raspberry  cpu is armhf like `cat /proc/cpuinfo` , but linux kernel is aarch64 like  `uname -a`  output .  
so the server tar ball show be https://dl.k8s.io/v1.20.0/kubernetes-server-linux-arm.tar.gz  not https://dl.k8s.io/v1.20.0/kubernetes-server-linux-arm64.tar.gz . is this right?
